HTML finish

SvistPetro opened the pull request.

SvistPetro committed:
HTML finish

SvistPetro committed:
HTML finish2

Yurii-Ilkevych commented on src/partials/task-eighteen.html:
```diff
@@ -1 +1,52 @@
+    type="text"
+    name="date-start"
+    id="date-start"
+    placeholder="Сьогодні"
```
"Форма бронювання готельного номеру містить в собі такі елементи форми як  <input> і <select>(яким необхідно додати мінімальну валідацію даних для введення за допомогою  атрибуту pattern)  та кнопку  “Оформити бронювання”. "
я так розумію потрібно в мадальне вікно добавити оції з можливістю вибору 3-номерів тобто має бути не плейсХолдер і можливість написання туди любої інфи, а має бути вибір номерів але не тільки двомісний люкс а і наступні два які ми маємо на сторінці
потім слідуюче дата бронювання має бути теж як <select> я так розумію але можливість вибору нехай мабуть буде тільки сьогодні згодом якщо буде час вернемось до цього питання та вияснимо як має бути реалізовано вибір дати бронюавання

Yurii-Ilkevych replied:
Атрибут pattern дозволяє вказати регулярний вираз (шаблон), щодо якого буде перевірятися значення поля. Використовується для розширення базової валідації. Наприклад, якщо потрібно, щоб ім'я користувача складалося з двох слів або пароль містив хоча б один символ у верхньому регістрі, один в нижньому регістрі і одне число.

Yurii-Ilkevych reviewed (commented):
"Форма бронювання готельного номеру містить в собі такі елементи форми як і (яким необхідно додати мінімальну валідацію даних для введення за допомогою атрибуту pattern) та кнопку “Оформити бронювання”. " я так розумію потрібно в мадальне вікно добавити оції з можливістю вибору 3-номерів тобто має бути не плейсХолдер і можливість написання туди любої інфи, а має бути вибір номерів але не тільки двомісний люкс а і наступні два які ми маємо на сторінці потім слідуюче дата бронювання має бути теж як я так розумію але можливість вибору нехай мабуть буде тільки сьогодні згодом якщо буде час вернемось до цього питання та вияснимо як має бути реалізовано вибір дати бронюавання

SvistPetro committed:
Final task 18

потрібно змінити шлях до svg-шок.
на закриття модалки в html
стрілки в селектах в CSS

SvistPetro committed:
Task18 update

виправив помилку
